src/data-sources/DataSourceConnector.ts

```diff
@@ -148,20 +148,25 @@ export class DataSourceConnector {
   static triggerAction(action: string, params: IStringDictionary, args: IDictionary) {
     var actionLocation = action.split(':');
 
-    if (actionLocation.length !== 2) {
+    if (actionLocation.length !== 2 && actionLocation.length !== 3) {
       throw new Error(`Action triggers should be in format of "dataSource:action", this is not met by ${action}`);
     }
 
     var dataSourceName = actionLocation[0];
     var actionName = actionLocation[1];
+    var selectedValuesProperty = "selectedValues";
```

Comment:
Let's talk about this - This makes sense, but I want to make sure I understand this, since it's an important change